.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-[type description  here]
+[TYPE DESCRIPTION HERE (SMART PEOPLE MAY REMOVE THIS LINE)]
```

Comment:
that's the english version of 'wer das liest ist doof'? :)

Reply:
That's what @okurz suggested. But I opted for a less offensive manner for sensitive people.